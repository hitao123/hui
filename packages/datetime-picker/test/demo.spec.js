describe('Datetimepicker component', () => {
  it('should show Datetimepicker', () => {

  });
});
