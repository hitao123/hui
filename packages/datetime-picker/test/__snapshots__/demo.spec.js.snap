// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div>
  <div></div>
</div>
`;
